src/Components/Authorization/LoginForm.tsx

```diff
@@ -0,0 +1,73 @@
+import { Formik } from "formik"
+import React from "react"
+import Yup from "yup"
+
+import {
+  BlockButton as Button,
+  ChangeMode,
+  FormContainer as Form,
+  inputValidators,
+  StyledFacebookButton,
+  StyledInput as Input,
+} from "./commonElements"
+import { FormComponentType } from "./Types"
+
+const LoginForm: FormComponentType = props => {
+  const { email: emailValidator } = inputValidators
+  return (
+    <Formik
+      initialValues={props.values}
+      onSubmit={props.handleSubmit}
+      validationSchema={Yup.object().shape({
+        email: emailValidator,
+        password: Yup.string().required("Please enter your password."),
```

Comment:
I did not reuse the password validator since this ticket raises the minimum password length from 6 to 8, and I figured it is best not to impose restrictions on what might be a password.

Reply:
@mzikherman - do we have any password requirements on the gravity side?